Lib/python/pyrun.swg

```diff
@@ -279,18 +283,26 @@ SwigPyClientData_New(PyObject* obj)
     /* the newraw method and newargs arguments used to create a new raw instance */
     if (PyClass_Check(obj)) {
       data->newraw = 0;
-      data->newargs = obj;
```

Comment:
Lest I face an accusation of frivolous change(!), I moved this so that each path where we were taking out an extra ref on `obj`, the `Py_INCREF(obj)` was before the operation which passed ownership of the reference to something else.

In this case, the ownership of this ref is being passed to `data->newargs`, but I found it easiest to understand the pattern of reference counting if the pattern of:

increment ref count -> assign ref to new owner

was followed throughout: I hope you will forgive me!